cmd/telemetry.go

```diff
@@ -27,6 +29,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const gaugeTelemetryEnabled = "GAUGE_TELEMETRY_ENABLED"
+
+var telemetryEnv = os.Getenv(gaugeTelemetryEnabled)
+
+func telemetryEnabled() bool {
+	if telemetryEnv == "" {
+		return config.TelemetryEnabled()
+	}
```

Comment:
This is because we print the message only if `GAUGE_TELEMETRY_ENABLED` is set. If we remove this check, there needs to be an additional check in Line-44.